src/XrdCms/XrdCmsCluster.cc

```diff
@@ -1800,7 +1808,17 @@ XrdCmsNode *XrdCmsCluster::SelbyLoad(SMask_t mask, XrdCmsSelector &selR)
                                   || (reqSS && np->isNoStage)))
               {selR.xFull = true; continue;}
            if (!sp) sp = np;
-              else{if (selR.needSpace)
+              else{
+                 if (weighed){
+                   //add 1 to the inverse load, this is to allow some selection in case reported loads hit 100
+                   weighed[i] = selCap + static_cast<int>(101 - np->myLoad +
+                     std::pow(101 - np->myLoad,
+                     scalingFactor)/2);
+                   selCap += static_cast<int>(101 - np->myLoad +
+                     std::pow(101 - np->myLoad,
+                     scalingFactor)/2);
```

Comment:
I find these split lines quite confusing to read, maybe it's better to just join the assignments for `weighed[i] =` and `selCap +=` into a single line. You are still computing `static_cast<int>(101 - np->myLoad + std::pow(101 - np->myLoad, scalingFactor)/2)` twice. You could just do this:
```suggestion
                   int nload = 101 - np->myLoad;
                   selCap += static_cast<int>(nload + std::pow(nload, scalingFactor)/2);
                   weighed[i] = selCap;
```

Reply:
thanks. I've applied that suggestion